scripts/globals/crafting.lua

```diff
@@ -140,17 +139,28 @@ xi.crafting.tradeTestItem = function(player, npc, trade, craftID)
     return newRank
 end
 
--- 1: test item
--- 2: skill point
--- 3: ??
--- 4: 0 Not in the guild 1 In the guild
--- 7: 0 Not in a guild already, 11: Multiple guilds
+-----------------------------------
+-- Guild Master event parameters
+-----------------------------------
+-- 1: Test item ID OR current vanadiel time.
+-- 2: Player REAL Skill at current guild craft.
+-- 3: Player REAL Skill cap at current guild craft.
+-- 4: Bitmask. First 9 bits used for guilds joined.
```

Comment:
If this is the case, and we're going to call out 3 of them here, then probably should comment all, or (preferably) comment the enum previously being 1-indexed vs these being 0.  _or_ (again) 0-index the table and update everything.